pkg/dependencies/factory.go

```diff
@@ -7,7 +7,12 @@ import (
 	"time"
 
 	"github.com/google/uuid"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
 
+	cpv1alpha1 "github.com/aws/eks-anywhere-packages/api/v1alpha1"
```

Comment:
nit: I would name this `packagesv1`. That kind of follows the naming we have been using for other api packages. The idea of not using `v1alpha1` is that it makes easier to bump versions (like to v1alpha2)